.github/workflows/cloud_chatops.yaml

```diff
@@ -1,16 +1,19 @@
-name: CI/CD Cloud ChatOps
+name: Cloud Chatops CI/CD
 
 on:
   push:
     branches:
       - master
   pull_request:
     paths:
-      - "cloud_chatops/**"
       - ".github/workflows/cloud_chatops.yaml"
+      - "cloud_chatops/src/**"
+      - "cloud_chatops/tests/**"
+      - "cloud_chatops/Dockerfile"
+      - "cloud_chatops/compose.yaml"
```

Comment:
Why do we get so specific? 
IMO: It should be anything in this folder triggers the workflow (better to be safe rather than sorry)

Reply:
👍